zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -126,19 +126,11 @@ object HttpData {
     }
   }
 
-  private[zhttp] final class UnsafeContent(private val httpContent: HttpContent) extends AnyVal {
-    def content: ByteBuf = httpContent.content()
-
-    def isLast: Boolean = httpContent.isInstanceOf[LastHttpContent]
-  }
-
-  private[zhttp] final class UnsafeChannel(private val ctx: ChannelHandlerContext) extends AnyVal {
-    def read(): Unit = ctx.read(): Unit
-  }
-
-  private[zhttp] final case class UnsafeAsync(unsafeRun: (UnsafeChannel => UnsafeContent => Unit) => Unit)
```

Comment:
Removing unnecessary wrapping from HttpContent and ChannelHandlerContext. Since we are being explicit about it being an "unsafe" API, the wrapping seems a bit unnecessary.